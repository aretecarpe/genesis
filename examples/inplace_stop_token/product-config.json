{
	"name": "inplace_stop_token_example",
	"version": "0.0.1",
	"description": "Example showcasing how to use inplace_stop_token",
	"type": "binary"
}