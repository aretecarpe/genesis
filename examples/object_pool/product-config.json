{
	"name": "object_pool_example",
	"version": "0.0.1",
	"description": "Example showcasing how to use object_pool",
	"type": "binary"
}