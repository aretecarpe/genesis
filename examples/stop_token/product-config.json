{
	"name": "stop_token_example",
	"version": "0.0.1",
	"description": "Example showcasing how to use stop_token",
	"type": "binary"
}