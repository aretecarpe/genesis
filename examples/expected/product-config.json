{
	"name": "exected_example",
	"version": "0.0.1",
	"description": "Example showcasing how one can use expected",
	"type": "binary"
}